hack/make/.build-rpm/docker-engine.spec

```diff
@@ -84,9 +84,12 @@ Requires: device-mapper >= 1.02.90-2
 %if 0%{?fedora} >= 22
 %global selinux_policyver 3.13.1-128
 %endif # fedora 22
-%if 0%{?centos} >= 7 || 0%{?rhel} >= 7 || 0%{?oraclelinux} >= 7
+%if 0%{?centos} >= 7 || 0%{?rhel} >= 7
 %global selinux_policyver 3.13.1-23
-%endif # centos,oraclelinux 7
+%endif # centos,rhel 7
+%if 0%{?oraclelinux} >= 7
+%global selinux_policyver 3.13.1-102.0.3.el7_3.15
```

Comment:
Oh hmm, it said:
```
          Requires: docker-engine-selinux >= 1.14.0-0.0.20170302.215905.git0e7aa9f.el7
           Removing: selinux-policy-targeted-3.13.1-102.0.2.el7.noarch (@anaconda/7.3)
               docker-engine-selinux
           Updated By: selinux-policy-targeted-3.13.1-102.0.3.el7_3.16.noarch (ol7_latest)
               Not found
```

Just doing a yum install `selinux-policy` gave me `selinux-policy-3.13.1-102.0.3.el7_3.16.noarch`
This works. Just strange it didn't hit the next version up.